packages/@aws-cdk/aws-cloudformation-codepipeline/README.md

```diff
@@ -1,24 +0,0 @@
-## AWS CodePipline Actions for AWS CloudFormation
```

Comment:
Looks like a merge issue. Probably needs to be merged into `aws-cloudformation`

Reply:
Sorry, I'm not following this comment 🤔This file has been removed (alongside the entire `aws-cloudformation-codepipeline` package). This text has been added into the `aws-cloudformation` README file ([here](https://github.com/awslabs/aws-cdk/pull/459/files#diff-9617ce3072a829e48678edc08172df15)).

Reply:
Oops you are right